chumweb/www/static/style.css

```diff
@@ -90,6 +90,12 @@ header, header:not(.section-site-info), .section-underline{
     padding-left: 0;
 }
 
+.link-list li a {
+    padding: 0.5em 0;
```

Comment:
This is exactly the same padding as before, resulting in the same non-spaced look, see screenshot in comment below.

Reply:
Was convinced it is fine by https://github.com/sailfishos-chum/sailfishos-chum.github.io/pull/28#issuecomment-2057749167.